### DISCUSSION TITLE
v1.3.0-beta.4

### DISCUSSION THREAD

**Discussion by EvanMulawski** [EvanMulawski/FanControl.CorsairLink#69] [Announcements]
- Fix: Fixes an issue introduced in v1.3.0-beta.3 that would stop the plugin from functioning when error detection notifications were enabled (the default) and multiple (10) errors were detected until the Fan Control dialog was dismissed.
- Logging: Shows a system message box instead of using the Fan Control dialog when error detection notifications are enabled (the default) and multiple (10) errors were detected.

<hr /><em>This discussion was created from the release <a href='https://github.com/EvanMulawski/FanControl.CorsairLink/releases/tag/v1.3.0-beta.4'>v1.3.0-beta.4</a>.</em>

**Comment by Fr0stX76** [comment]
@EvanMulawski 

With this version I get the following in my log

[WRN] Device Enumeration: Failed to enumerate SiUsbXpress devices

I assume it is because I did not install the SiUsbXpress driver (as I do not need it for any device). Should I have installed it to get rid of it? or should I just ignore the warning altogether.

Probably just my OCD but I kind of find it annoying to see a warning lol. Maybe this warning would be better off as an info instead of a warning of failure when there is no device to detect. No big deal but I tough Id mention It.

**Reply by EvanMulawski** [reply]
It can be ignored if you don't have devices that need it. I think it's fine as a warning as it depends on the user's intent and might be important to someone.

**Reply by Fr0stX76** [reply]
Gotcha. Ill just ignore this one
